core/domain/stats_services.py

```diff
@@ -295,6 +295,28 @@ def save_stats_model_transactional(exploration_stats):
         _save_stats_model, exploration_stats)
 
 
+def get_view_multi(exploration_ids, version_numbers):
```

Comment:
This used to be get_views_multi(); keep the naming.

Reply:
EDIT: superseded by https://github.com/oppia/oppia/pull/4201#discussion_r156903427

Reply:
Redundant now.